react/src/components/ImageEnvironmentSelectFormItems.tsx

```diff
@@ -695,39 +716,98 @@ const ImageEnvironmentSelectFormItems: React.FC<
+                                              keyword={versionSearch}
+                                              key={tag.key}
+                                            >
+                                              {tagAlias(tag.key)}
+                                            </TextHighlighter>
+                                          ),
+                                          color: isCustomized ? 'cyan' : 'blue',
+                                        },
+                                        {
+                                          label: (
+                                            <TextHighlighter
+                                              keyword={versionSearch}
+                                              key={tagValue}
+                                            >
+                                              {tagValue}
+                                            </TextHighlighter>
+                                          ),
+                                          color: isCustomized ? 'cyan' : 'blue',
+                                        },
+                                      ]}
+                                    />
+                                  );
+                                },
+                              )}
+                            </Flex>
                           </Flex>
-                          <Flex
-                            direction="row"
-                            // set specific class name to handle flex wrap using css
-                            className={
-                              isDarkMode ? 'tag-wrap-dark' : 'tag-wrap-light'
-                            }
-                            style={{
-                              marginLeft: token.marginXS,
-                              flexShrink: 1,
-                            }}
-                          >
-                            {requirementTags || '-'}
+                        ) : (
+                          <Flex direction="row" justify="between">
+                            <Flex direction="row">
+                              <TextHighlighter keyword={versionSearch}>
+                                {version}
+                              </TextHighlighter>
+                              <Divider type="vertical" />
+                              <TextHighlighter keyword={versionSearch}>
+                                {metadataTagAlias}
+                              </TextHighlighter>
+                              <Divider type="vertical" />
+                              <TextHighlighter keyword={versionSearch}>
+                                {image?.architecture}
+                              </TextHighlighter>
+                            </Flex>
+                            <Flex
+                              direction="row"
+                              // set specific class name to handle flex wrap using css
```

Comment:
The CSS contents are same, do we need to split them?

[ref](https://github.com/lablup/backend.ai-webui/blob/188d31f8bd994635ee8e24dfc108663041a5d442/react/src/components/ImageEnvironmentSelectFormItems.css#L15-L39)

Reply:
what did you mean?